source/features/quick-repo-deletion.tsx

```diff
@@ -21,13 +21,19 @@ function handleToggle(event: delegate.Event<Event, HTMLDetailsElement>): void {
 		'.rgh-open-prs-of-forks' // PRs opened in the source repo
 	]);
 
-	if (hasContent && !confirm('This fork has open issues/PRs, are you sure you want to delete everything?')) {
+	if (hasContent && !confirm('This repo has open issues/PRs, are you sure you want to delete everything?')) {
 		// Close the <details> element again
 		event.delegateTarget.open = false;
-	} else {
-		// Without the timeout, the same toggle event will also trigger the AbortController
-		setTimeout(start, 1, event.delegateTarget);
+		return;
+	}
+
+	if (!pageDetect.isForkedRepo() && !confirm('⚠️ ⚠️ ARE YOU TOTALLY SURE YOU WANT TO DELETE THIS REPOSITORY? ⚠️ ⚠️')) {
```

Comment:
Heh this looks comical… more professional suggestions welcome before this is merged

Reply:
How about mimicking the GitHub message?
```
This action cannot be undone. This will permanently delete the repository,
wiki, issues, comments, packages, secrets, workflow runs, and remove all
collaborator associations.
```

Reply:
Good. I’d preserve a single emoji at the beginning